src/cargo/core/compiler/build_runner/compilation_files.rs

```diff
@@ -88,9 +89,15 @@ impl fmt::Debug for UnitHash {
 /// mangled symbols need to be stable between different builds with different
 /// settings. For example, profile-guided optimizations need to swap
 /// `RUSTFLAGS` between runs, but needs to keep the same symbol names.
+///
+/// [`Metadata::host_independent_hash`] is a simple hash that excludes any metadata
```

Comment:
We'll need to evaluate how the information at https://doc.rust-lang.org/nightly/nightly-rustc/cargo/core/compiler/fingerprint/index.html#fingerprints-and-unithashs will need to be updated based on this